### PR TITLE
TRAD-499 Drift Orderbook GET and stream support

### PR DESCRIPTION
**Pull request by DimaCantemir**
TRAD-499 Drift Orderbook GET and stream support

### CODE REVIEW

File: bxsolana/provider/base.ts [hunk 1]
```diff
@@ -91,6 +116,76 @@ export abstract class BaseProvider implements Api {
         }
     }
 
+    postPerpOrder(
```
Comment (aspin) [inline bxsolana/provider/base.ts:119]: think you also need the following:

```
submitPerpOrder
submitClosePerPositions
submitCreateUser
submitDepositCollateral
submitWithdrawCollateral
```

File: examples/index.ts [hunk 2]
```diff
@@ -178,13 +178,13 @@ async function grpc() {
         await doAmmStreams(provider)
     }
 
-    if (runLongExamples) {
+    /*if (runLongExamples) {
```
Comment (aspin) [inline examples/index.ts]: uncomment (I'd suggest creating a run config in your IDE that has this environment variable set so we don't accidentally commit code like this)

File: examples/index.ts [hunk 3]
```diff
@@ -317,10 +322,6 @@ async function doStreams(provider: BaseProvider) {
     console.info(" ")
     console.info(" ")
 
-    await callGetMarketDepthStream(provider)
```
Comment (aspin) [inline examples/index.ts:320]: undelete?

**Review by aspin — CHANGES_REQUESTED** [review]

File: examples/index.ts [hunk 4]
```diff
@@ -121,13 +121,13 @@ async function http() {
     console.info(" ----  HTTP Amm Requests  ----")
     await doAmmRequests(provider)
 
-    if (runLongExamples) {
+    /* if (runLongExamples) {
```
Comment (aspin) [inline examples/index.ts]: one more here

**Review by aspin — APPROVED** [review]

File: bxsolana/provider/base.ts (hunk 1 above)
Comment (DimaCantemir) [reply bxsolana/provider/base.ts:119]: this is Mahmoud's part

**Pull request merged**